staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/accessors.go

```diff
@@ -66,7 +66,7 @@ type mutatingWebhookAccessor struct {
 }
 
 func (m mutatingWebhookAccessor) GetUID() string {
-	return m.Name
+	return m.uid
```

Comment:
Yikes, glad I saw this before we released.

Reply:
can you add a quick unit test for mutatingWebhookAccessor and validatingWebhookAccessor that verifies the uid passed into `New...` is what is returned